bokehjs/src/coffee/common/pgrid.coffee

```diff
@@ -0,0 +1,109 @@
+$ = require "jquery"
+_ = require "underscore"
+Backbone = require "backbone"
+build_views = require "./build_views"
+ContinuumView = require "./continuum_view"
+HasProperties = require "./has_properties"
+{logger} = require "./logging"
+widget = require "phosphor-widget"
+gridpanel = require "phosphor-gridpanel"
+messaging = require "phosphor-messaging"
+
+
+class PGridView extends ContinuumView
+  #className: ""
+
+  initialize: (options) ->
+    super(options)
+    window.box = this
+    @views = {}
+    
+    @panel = new gridpanel.GridPanel()
+    @panel.node.style.width = '100%'
+    @panel.node.style.height = '100%'
+    @panel.node.style.position = 'absolute'
+    @el.appendChild(@panel.node)
```

Comment:
@sccolbert This is where the widget is set to fill the screen. The widget is put in the correct position in the DOM, but we just make it to fill the whole space, which would not be right if its not the toplevel widget/layout. I consider this a side-effect of using phosphor for one particular widget. If we go all phosphor, the panel would be attached to its parent phosphor panel (or to the root DOM element).